qemu/tests/cfg/nvdimm.cfg

```diff
@@ -100,12 +100,13 @@
         - nvdimm_negative:
             type = nvdimm_negative
             start_vm = no
+            vm_mem_limit = 8G
             size_mem = 8G
             pre_command = "dd if=/dev/zero of=${nv_backend} bs=1M count=8192"
             variants:
                 - align_max:
                     align_mem = 4G
-                    error_msg = "the alignment \(0x100000000\) (is not supported|exceeds the expected maximum alignment)"
+                    error_msg = "the alignment (0x100000000) exceeds the expected maximum alignment, memory will get fragmented and not all 'maxmem' might be usable for memory devices."
```

Comment:
Is this a new change with new product version ?

Reply:
@yanan-fu yes, if for example we take a look to qemu 4.2.0 using the same qemu-kvm cmd
```bash
QEMU 4.2.0 monitor - type 'help' for more information
(qemu) qemu-kvm: -device nvdimm,id=dimm-mem1,memdev=mem-mem1: the alignment (0x100000000) is not supported
```

Reply:
All the versions need to be compatible

Reply:
@yanan-fu updated now to be compatible with the previous error message (valid until qemu-kvm-6.0.0) and the current one

**QEMU version**
```bash
/usr/libexec/qemu-kvm --version
QEMU emulator version 4.2.0 (qemu-kvm-4.2.0-60.module+el8.5.0+14545+9e40c7b1.2)
```
**Test results**
```bash
 (1/2) Host_RHEL.m8.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.nvdimm.nvdimm_negative.align_max.q35: STARTED
 (1/2) Host_RHEL.m8.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.nvdimm.nvdimm_negative.align_max.q35: PASS (7.95 s)
 (2/2) Host_RHEL.m8.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.nvdimm.nvdimm_negative.align_illegal.q35: STARTED
 (2/2) Host_RHEL.m8.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.nvdimm.nvdimm_negative.align_illegal.q35: PASS (7.72 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
```
**QEMU version**
```bash
/usr/libexec/qemu-kvm --version
QEMU emulator version 6.0.0 (qemu-kvm-6.0.0-33.module+el8.5.0+14188+8c5ecfdd.3)
```
**Test results**
```bash
 (1/2) Host_RHEL.m8.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.nvdimm.nvdimm_negative.align_max.q35: STARTED
 (1/2) Host_RHEL.m8.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.nvdimm.nvdimm_negative.align_max.q35: PASS (37.95 s)
 (2/2) Host_RHEL.m8.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.nvdimm.nvdimm_negative.align_illegal.q35: STARTED
 (2/2) Host_RHEL.m8.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.nvdimm.nvdimm_negative.align_illegal.q35: PASS (7.68 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
```